// function outside of callback logic

function onOrderError(error) {
  return `Error! ${error}`;
}

console.log(onOrderError('treść błędu'))